### DISCUSSION TITLE
Weird defect on bottom of prints

### DISCUSSION THREAD

**Discussion by FlaMtnBkr** [OrcaSlicer/OrcaSlicer#6488] [General]
I'm using a smooth G10 print bed which makes the bottom of the print easy to see and I'm having a weird defect that hasn't always been there, though I'm not real sure when it started. 

The wall perimeter prints fine and looks great but the infill is messed up and creates (roughly) lines perpendicular to the print lines so it doesn't seem completely random. I've been using v2.0.0 and just updated to 2.1.1 and a small square made the infill look like the print lines were split into 2 towards the middle. This made me think of the "Infill Combination" where layers are combined so I unchecked it and printed it again. This time the defect was more random but still there. And again, the perimeter looks fine so it seems like it's something with the software. I think? 

Any thoughts? Or settings I should look at? I've posted this elsewhere but it didn't seem like others were having it so maybe it's me combination of settings? Though my main settings haven't changed in a while except minor tweaks or when new features are added.

I'd love to get the bottoms looking good again so let me know if I can provide any more info. 

Thanks! 

![IMG_20240809_152959189~2](https://github.com/user-attachments/assets/7483d870-f7eb-4165-9581-f92d3ac11986)

**Comment by FlaMtnBkr** [comment]
Hopefully I'll get some input to the above but in the meantime does anyone know the oldest version that still works correctly? I've tried using default process presets but still get the print defect which I didn't get with the first version of software I used. So I'm thinking the defect is being caused by a new setting that wasn't in that version. So using an older version of software should let me print and get a nice bottom layer which would be great to have in the parts in currently trying to print.

The first version I used was v1.7.0 and it doesn't appear to be working correctly. It won't let me login and none of my presets are available including my modified printer settings that doesn't push down on the tab along the back of Bambu build plates, as well as user filament and process presets. I can probably go thru and modify everything and hope it saves it correctly but would rather just download the oldest version that will use what's stored on my computer. 

That being said, I guess another issue is that the "portable" versions of the software are still not truly portable. They don't save settings to the folder the program is "installed" or unzipped to. I've tried seeing how other people deal with this including using "-datadir" in the shortcut but this just resulted in not having my presets without knowing where they are saved and which are relevant to copy to the folder the shortcut points to. It would be nice if there was a place in settings or preferences where you could choose where to save user profiles whether it's a local or network drive. And then maybe a script or application that checks if this value is modified and if it is, copies the relevant files to that location so people like me don't have to stumble thru trying to figure it out. 

I guess I should try Bambu Studio it another slicer software to see if that fixes things and allow me to print until I can get this issue sorted.

Sorry for rambling and I'll update if different software does indeed fix the defect.

**Comment by Ergonomicmike** [comment]
Since you asked if "any thoughts," my initial thought is that your Z-offset is too low and/or there's something wrong with your extruder path. (Causing intermittent over and under extrusion.) That is, this doesn't look like a software bug to me. (Altho I can't explain why the perimeters look fine. But they are vertical and horizontal - whereas the infill is diagonal. So perhaps a clue there as to a mechanical problem? (Core XY? Power supply droop when both steppers are turning?))

As for portable versions storing data - yeah, a pet peeve of mine from a long time ago when M$ opted to use a Registry for everything in Windows. (As opposed to .ini files.) I suppose, on the other hand, that the advantage of storing the settings for the portable version in Windows\Users is that I don't have to copy a "user data" folder every time that I update Orca Portable. (Although the Users folder grows without bound with each never verison of Orca portable.)

**Comment by FlaMtnBkr** [comment]
I'm using a Bambu P1S and there is no way to adjust the Z-offset (as far as I know) and I've checked the filament path and haven't found anything that seems like a restriction. I've also tried rotating the part 45* as well as rotating the infill 45* so that the infill is printed in a straight line and the perimeter is diagonal and it didn't change anything.

I've tried messing with every setting I can think of and the one that makes the biggest difference is lowering the "Bottom Surface Flow Ratio" in the Quality tab. It reduces the majority of the defects so that the beginning and end of the infill lines look pretty good about like the perimeter lines, but then appear to split into 2 narrow lines instead of being a solid line (can see the beginning and end being better a little in the above picture). So even though it's better, there is still something going on and some prints are worse, though so far not as bad as the above picture.

I've tried different filaments/material types as well as messing with almost every setting and going thru all the calibrations and still haven't figured anything out. The biggest thing I don't understand is why the perimeter lines print fine and the infill doesn't?! I've made the speed equal for both which is pretty slow at 40 mm/s and the reason I still think it's something software related? It just seems like the perimeter would have problems if it was something mechanical or the filament and there would be problems with the rest of the print which there isn't. If the whole bottom printed like the perimeter lines always do, it would be great.

If anyone has any suggestions of things to try it would be greatly appreciated! I have been out of town and haven't been able to troubleshoot much more than adjusting settings but I'll be changing out hardware (nozzle/tubing/etc), trying different software, and putting in a support ticket with Bambu as I have the time.

Edit: I put a new 0.6mm nozzle in and a new section of PTFE tubing from the back of the printer to the hotend and it's printing the same way so far...perimeter is fine and the beginning and end of the infill lines are good for about 6 mm and then the defects begin. The first couple prints after the change have the defect where the line splits into what looks like 2 small lines running parallel. I'll keep trying things and I'll update if I ever figure something out. I just can't imagine 2 long infill lines having the same defect in roughly the same spot so if creates a perpendicular line of defects if it was truly random. Something is causing a defect in the same spots of each line which isn't random. A hardware or filament problem would be random. The only thing that wouldn't be random is something the software is doing, I think...

**Comment by FlaMtnBkr** [comment]
Still haven't figured anything out. Have installed a couple more nozzles and new PTFE tubing.

Here is a temp tower with a brim. Don't know why half of the brim is ok but the inner part isn't, but something is obviously wrong and I can't imagine it being anything besides the software...

![IMG_20240909_194404441~2](https://github.com/user-attachments/assets/a5a88f59-5b0b-42a2-b3b7-dbbbe5d51e4f)

**Comment by Ergonomicmike** [comment]
I'm still thinking that this is a hardware problem, subtle as it might be. Have you asked Bambu if they've seen this before?

Does it do this with a pre-sliced file that came with your printer or gcode that someone has posted for your model printer?

Have you tried a different slicer to eliminate that it's Orca causing this problem?

As a last brainstorming thought, temperature wandering during infill?

If you knew someone else with a Bambu, they could print your gcode and see if they have this same result.

**Comment by FlaMtnBkr** [comment]
I've been back and forth with Bambu CS too many times to count and the only thing they came up with was sending me a new heating bed which did nothing. I ask their reasoning and they said that mine might not be flat/level even though they told me every time I sent in the printer logs that the Z offset was normal and within their tolerances. Which says to me they were grasping at straws. I wish they would have sent a new print plate instead because I could at least use that...

It's been close to 2 months and my ticket is still open because they never came up with anything that worked. Since then, I tried printing a model and increased the initial layer thickness to 0.3 as it was suggested to get better bed adhesion. What I found was that it made things worse. After thinking about it, the thickness changed but the speed didn't which meant the flow rate had to increase. I first tried making the first layer thinner (0.16) which helped and then tried even thinner @ 0.12 which further improved things.

After that, I tried reducing the first layer flow ratio and went down to 0.90 and even 0.80. Between the 2 settings being reduced, I got the first layer to print fairly great and had to adjust the 2 depending on the filament. I also went back and did flow rate calibrations for each filament and set them to the best looking top layer and still the reduction in those 2 settings is the only thing that made the first layer look good. 

I have no idea why making the first layer thinner and reducing the flow ratio is what has solved this problem. It seems like if anything they would need to be increased to overcome resistance in the printer yet lowering them is what made the difference. I would also think this rules out a hardware issue? I still have lots of experimenting if I want to try to figure this out and make sure it always works. But it's essentially fixed the problem and I figured I would update this as lowering those 2 settings quite a bit lower than normal would be the last thing I would have thought and I just happened to get lucky at this point.

I will keep making changes and try to figure things out and will update again if I find something. I still can't believe I'm the only one that has run into this and don't know if most people don't use a flat/smooth build plate which makes it more obvious? And just to reiterate, I've tried different rolls and types of filaments, as well as installing new PTFE tubing and complete nozzle. As well as a new heat bed that Bambu sent me. 

Anyways, this is where I'm at and no hardware that I changed helped, just a reduction in layer height and flow ratio for the first layer only. The rest prints normal...I'll keep trying different combinations but at this point I've got a good looking bottom layer/ print though I have no clue why a change in software settings has fixed it so far...
